src/politodown/polito.py

```diff
@@ -70,9 +70,26 @@ async def get_videostores(year: int) -> dict[str, Videostore]:
                 continue
 
             videolesson_name = videolesson.text.strip()
-            cor, = data_regex.search(videolesson["href"]).groups()
-            videolessons[videolesson_name] = \
+            if data_regex.match(videolesson["href"]).group(1):
+                cor = data_regex.search(videolesson["href"]).group(2)
```

Comment:
`data_regex.match(videolesson["href"])` is already used in line 65. What if we would use a variable for it?